### DISCUSSION TITLE
How to use earlystopping in deepCTR-Torch ?  能用earlystopping吗,咋用？

### DISCUSSION THREAD

**Discussion by CFZhai** [shenweichen/DeepCTR-Torch#268] [Q&A] [answered]
An detailed example will be appreciated!

**Comment by CFZhai** [comment]
自问自答。

**Reply by CFZhai** [reply]
> just add a TF instance of EarlyStopping.

early_stopping = tf.keras.callbacks.EarlyStopping( monitor="val_mse", patience=10,  ) #restore_best_weights=True

model.fit(train_model_input,train[target].values,batch_size=32,epochs=1000,verbose=2,validation_split=0.1, callbacks= [early_stopping])

**Comment by zanshuxun** [comment, marked as answer]
https://deepctr-torch.readthedocs.io/en/latest/FAQ.html#set-learning-rate-and-use-earlystopping
